docs/assets/css/secondary-nav.less

```diff
@@ -1,85 +1,178 @@
-                     unit(@grid_gutter-width / @font-size-small, em);
-            border-bottom-style: solid;
-            border-width: 0; // Needed to override button styles
-            border-bottom-width: 1px;
-            background: @gray-10;
-            font-size: unit(@font-size-small / @base-font-size-px, em);
-            font-weight: 500;
-            .u-link__colors(@dark-gray, @dark-gray, @dark-gray, @dark-gray, @dark-gray,
-                            @gray-20, @gray-20, @gray-20, @gray-20, @gray-20);
-            text-align: left; // Needed to override button styles
+    &.o-expandable__background {
+        .respond-to-min(901px, {
+            background: transparent;
         });
+    }
 
-        .respond-to-max(600px, {
-            padding-left: unit((@grid_gutter-width / 2) / @font-size-small, em);
-            padding-right: unit((@grid_gutter-width / 2) / @font-size-small, em);
-        });
+}
+
+.nav-secondary {
+    ul > li > .h5 {
+        color: #676A6F;
+        font-weight: 500;
+    }
+    .m-list__links {
 
-        .respond-to-min(801px, {
-            display: inline-block;
-            padding: unit(5px / @font-size-large, em)
-                     unit(15px / @font-size-large, em);
-            border-left-style: solid;
-            border-left-width: 5px;
-            font-size: unit(@font-size-large / @base-font-size-px, em);
-            font-weight: 500;
-            .u-link__colors(@pacific, @pacific, @black, @black, @black,
-                            transparent, transparent, @green, @green, @green);
+        .respond-to-min(901px, {
+                margin-top: unit( 5px / @base-font-size-px, em );
         });
 
-        &__current {
-            .respond-to-max(800px, {
-                display: none;
+        .m-list_item {
+            .respond-to-min(901px, {
+                margin-bottom: unit( 4px / @base-font-size-px, em );
+            });
+        }
+
+        .m-list_link {
+            padding-left: unit( 20px / @base-font-size-px, em );
+            .respond-to-max(900px, {
+                border-style: solid;
+                border-color: @gray-40;
             });
-            .respond-to-min(801px, {
-                .u-link__colors(@black, @black, @black, @black, @black,
-                                @green, @green, @green, @green, @green);
+            .respond-to-min(@bp-sm-min, {
+                margin-left: unit( 20px / @base-font-size-px, em );
+                padding: unit( 4px / @base-font-size-px, em ) unit( 15px / @base-font-size-px, em );
+                border-bottom: none;
+                color: @black;
+                &:hover,
+                &:focus {
+                    background: @gray-10;
+                }
             });
         }
+    }
+}
+
+
+//
+// Mixin: Bleedbar sidebar (on left) styling
+// @breakpoint argument: pixel value of minimum screen width at which you want the bleedbar styling to appear
+//
+
+.content__bleedbar_left {
```

Comment:
So this is saying on the `content` block, have a `bleedbar` modifier, which itself has a `left` element. However, as mentioned in chat I believe this should be `content__bleedbar-left`, because it does not appear you are modifying an element of `bleedbar`, but instead creating a new modifier, `bleedbar-left`.